For NetBSD, force pass thread configure and set USE_AMD64_IOPL

--- configure.orig	2021-07-29 18:48:26 UTC
+++ configure
@@ -12065,7 +12065,7 @@ echo 'failed to copy INSTALL from util-m
 
 
 case $host_os in
-    solaris*)
+    nothing*)
         # Solaris 2.0 - 11.3 use SysV man page section numbers, so we
         # check for a man page file found in later versions that use
         # traditional section numbers instead
@@ -21844,7 +21844,7 @@ if test `${RAWCPP} < conftest.$ac_ext |
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
 $as_echo "no" >&6; }
 else
-	if test `${RAWCPP} -traditional < conftest.$ac_ext | grep -c 'preserve   \"'` -eq 1 ; then
+	if true; then
 		TRADITIONALCPPFLAGS="-traditional"
 		RAWCPPFLAGS="${RAWCPPFLAGS} -traditional"
 		{ $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
@@ -23213,9 +23213,14 @@ $as_echo "#define USE_ALPHA_PIO 1" >>con
 	esac
 	GLX_ARCH_DEFINES="-D__GLX_ALIGN64 -mieee"
 	;;
-  arm*)
+  arm*|aarch64*)
 	ARM_VIDEO=yes
 	DEFAULT_INT10="stub"
+	case $host_os in
+		*freebsd*)
+			$as_echo "#define USE_DEV_IO 1" >>confdefs.h
+		 ;;
+	esac
 	;;
   i*86)
 	I386_VIDEO=yes
@@ -23255,7 +23260,7 @@ $as_echo "#define USE_DEV_IO 1" >>confde
 $as_echo "#define USE_DEV_IO 1" >>confdefs.h
  ;;
 		*netbsd*)
-$as_echo "#define USE_I386_IOPL 1" >>confdefs.h
+$as_echo "#define USE_AMD64_IOPL 1" >>confdefs.h
 
 				SYS_LIBS=-lx86_64
 				;;
@@ -25069,7 +25074,8 @@ case ${host_os} in
 	netbsd*)
 	# use libc stubs, don't link against libpthread, to allow
 	# dynamic loading
-	ax_pthread_flags=""
+	ax_pthread_flags="none"
+	ax_pthread_ok="yes"
 	;;
 esac
 
